ska_bin_starcheck

```diff
@@ -0,0 +1,5 @@
+ #!/bin/bash
+
+SKA_bin=`dirname $0`
+eval `${SKA_bin}/flt_envs -shell sh`
+exec ${SKA_ARCH_OS}/bin/starcheck "$@"
```

Comment:
Here, I've set up basically a double wrapper.  This very light wrapper calls the starcheck wrapper in SKA_ARCH_OS.   I don't know if the double-wrapper idea makes more or less sense than having a "make install" step that copies a more full wrapper into SKA/bin.